src/tests/ftest/ior/crash_ior.yaml

```
@@ -35,7 +35,7 @@ ior:
   transfersize_blocksize:
     4K:
       transfer_size: '4K'
-      block_size: '4G'
+      block_size: '10G'
```

Since we're using stonewalling 100s and want to kill a rank during IO, I think the idea with this IOR config is for the block size to be large enough that it takes 100s to write it. In that case, we can just increase the block size to something very large to account for any performance variation, since the actual bytes written will be limited by stonewall 100s. I.e. we won't actually write the full block_size.
```suggestion
      block_size: '100G'
```